plugins/typo_squatter/plugin.spec.yaml

```diff
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: typo_squatter
 title: Typo Squatter
 description: Detect cybersquatting of domains to aid in phishing investigation and analysis
-version: 1.0.1
+version: 1.0.2
+supported_versions: ["CertStream 1.12"]
```

Comment:
Supported Versions are supposed to list versions of vendor software that a plugin directly associates with.
We do not need to specify a version of CertStream for the Typo Squatter plugin because the CertStream package is an upstream dependency, not a direct association of this plugin.